From 0edd10050a6ec13384381d28090d11e3d4bcfff1 Mon Sep 17 00:00:00 2001
From: Nhan Nguyen <nhan.nguyen.yb@renesas.com>
Date: Mon, 4 Apr 2022 12:00:15 +0700
Subject: [PATCH 2/2] dts: renesas: r9a07g043f01-{dev,smarc}: set cma mem size

Set CMA mem size the value 0x08000000 for both RZ/Five Dev board and
SMARC board.

Signed-off-by: Nhan Nguyen <nhan.nguyen.yb@renesas.com>
---
 arch/riscv/boot/dts/renesas/r9a07g043f01-dev.dts   | 13 +++++++++++++
 arch/riscv/boot/dts/renesas/r9a07g043f01-smarc.dts |  2 +-
 2 files changed, 14 insertions(+), 1 deletion(-)

diff --git a/arch/riscv/boot/dts/renesas/r9a07g043f01-dev.dts b/arch/riscv/boot/dts/renesas/r9a07g043f01-dev.dts
index 5e8f8a14d53e..169c254a903b 100755
--- a/arch/riscv/boot/dts/renesas/r9a07g043f01-dev.dts
+++ b/arch/riscv/boot/dts/renesas/r9a07g043f01-dev.dts
@@ -20,4 +20,17 @@ memory@48000000 {
 		/* first 128MB is reserved for secure area. */
 		reg = <0 0x48000000 0 0x38000000>;
 	};
+
+	reserved-memory {
+		#address-cells = <2>;
+		#size-cells = <2>;
+		ranges;
+
+		reserved: linux,cma@58000000 {
+			compatible = "shared-dma-pool";
+			linux,cma-default;
+			reusable;
+			reg = <0x0 0x58000000 0x0 0x08000000>;
+		};
+	};
 };
diff --git a/arch/riscv/boot/dts/renesas/r9a07g043f01-smarc.dts b/arch/riscv/boot/dts/renesas/r9a07g043f01-smarc.dts
index 5632fc8be881..56cbabc5b32c 100755
--- a/arch/riscv/boot/dts/renesas/r9a07g043f01-smarc.dts
+++ b/arch/riscv/boot/dts/renesas/r9a07g043f01-smarc.dts
@@ -42,7 +42,7 @@ reserved: linux,cma@58000000 {
 			compatible = "shared-dma-pool";
 			linux,cma-default;
 			reusable;
-			reg = <0x0 0x58000000 0x0 0x10000000>;
+			reg = <0x0 0x58000000 0x0 0x08000000>;
 		};
 	};
 };
-- 
2.25.1

